lessonbuilder/tool/src/webapp/css/Simplepagetool.css

```diff
@@ -198,7 +198,7 @@ div.action span .nextbutton, .Mrphs-siteHierarchy span .nextbutton{
 
 }
 .act .deleteButton {
-    float: none
+    float: right
```

Comment:
Now that it looks like a red button, will that help/resolve your concern, @plukasew ?